### ISSUE TITLE
How to find out the size of a LXC container?

### ISSUE THREAD

**Issue by nomadlena** [lxc/lxc#912]
Hello, I've searched online for a long time and haven't seen anything commands that can show a container's size. I know lxc-info command shows a bunch of information about a running container like CPU use, memory use, BlkIO use etc., but how to find out the size(i.e. disk use) of a container? If some larger applications are running inside the container, surely it would take up some disk space when the container is running. Which command can be used to find out that information?
 
Thank you!!

**Comment by Kubuxu** [comment]
It isn't that easy of a question to answer. In case of normal (EXT) filesystem you should be able to just check size of rootfs of the container.

But in case of CoW filesystems (ZFS, btrfs) the directory size gives you no information whatsoever.

It is because on CoW filesystem new containers might be created taking only few megabytes of additional space. Rest is just shared between them.

**Comment by nomadlena** [comment]
Hello Kubuxu, thanks for your comment. According to this post "The filesystem can be set with --fstype (the default is “ext4”)" from Stéphane Graber, my container should be using ext filesystem by default. Then it should be easy to check the size of the container. However I found that the rootfs of my container is only 4.0k and when I listed out the things in that directory, everything inside is also 4.0k. So this means the rootfs cannot reflect correctly the size of my container even if the default is ext4 type. Why? Thank you!!
